src/etl/molecular_interaction_etl.py

```diff
@@ -118,23 +129,30 @@ def populate_genes(self):
 
         return master_gene_set
 
-    def query_crossreferences(self, crossref_prefix):
-        query = "MATCH (g:Gene)-[C:CROSS_REFERENCE]-(cr:CrossReference) WHERE cr.prefix = {parameter} RETURN g.primaryKey, cr.globalCrossRefId"
+    @staticmethod
+    def query_crossreferences(crossref_prefix):
+        '''Query Cross References'''
+
+        query = """MATCH (g:Gene)-[C:CROSS_REFERENCE]-(cr:CrossReference)
+                   WHERE cr.prefix = {parameter}
+                   RETURN g.primaryKey, cr.globalCrossRefId"""
         return Neo4jHelper().run_single_parameter_query(query, crossref_prefix)
 
     def populate_crossreference_dictionary(self):
-        # We're populating a rather large dictionary to use for looking up Alliance genes by their crossreferences.
-        # Edit the list below if you'd like to add more crossreferences to the dictionary.
-        # The key of the dictionary is the crossreference and the value is the Alliance gene to which it resolves.
+        ''' We're populating a rather large dictionary to use for looking up Alliance genes by
+            their crossreferences.
+            Edit the list below if you'd like to add more crossreferences to the dictionary.
+            The key of the dictionary is the crossreference and the value is the Alliance
+            gene to which it resolves.'''
 
         master_crossreference_dictionary = dict()
 
         master_crossreference_dictionary['UniProtKB'] = dict()
         master_crossreference_dictionary['ENSEMBL'] = dict()
         master_crossreference_dictionary['NCBI_Gene'] = dict()
 
-        for key in master_crossreference_dictionary.keys():
-            logger.info('Querying for %s cross references.' % (key))
+        for key in master_crossreference_dictionary:
```

Comment:
Not sure .keys() is the same as key in master_crossreference_dictionary? @sierra-moxon